lib/features.json

```diff
@@ -298,10 +298,5 @@
     "name": "entityNameLowerCaseNoUnderscore",
```

Comment:
this commit broke the tests

why are you undoing that ?

Reply:
theres no reason to make this change if as soon as 1.17.1 the packet is literally back to what it was

Reply:
fixed

Reply:
can we assume no server will be running 1.17 and all 1.17.1 ?

Reply:
no lol, why would we just leave a bug like that

Reply:
no as in we can't assume it ?
this revert introduces a bug for 1.17
we can choose not to support 1.17 and only 1.17.1 if nobody uses 1.17

Reply:
1. How does it introduce a bug? 
2. Why would people skip 1.17?

Reply:
1. because 1.17 has the different entity destroy packet which you just removed support for
2. I don't know, that's what I'm asking you. If they don't skip 1.17, then this commit shouldn't be done

Reply:
ah looking at the code I see you actually kept support for 1.17 , didn't read in detail before
all good then

Reply:
Yeah I was very confused